Update git webhook triggers troubleshooting section (#5712)
harness/developer-hub: docs/platform/triggers/triggering-pipelines.md

- Check the execution history (select **Execution History** in the top right of the Pipeline Studio).
- Verify that the runtime inputs are correct.
- Check the payloads sent from the Git provider and compare the relevant fields with your trigger conditions. For example, in GitHub you can view the full payload of each event sent from a specific webhook.

> - If you are using a self-hosted Git server, check if your server is able to make https requests to the Harness Git webhooks URL `https://app.harness.io/gateway/ng/api/webhook`.

Reviewer: ```suggestion
- If you are using a self-hosted Git server, check to be sure your server can make HTTPS requests to the Harness Git webhooks URL `https://app.harness.io/gateway/ng/api/webhook`.
```
